htdocs/fourn/commande/list.php

```diff
@@ -1536,7 +1537,15 @@
 		}
 		// Status
 		if (!empty($arrayfields['cf.fk_statut']['checked'])) {
-			print '<td class="right nowrap">'.$objectstatic->LibStatut($obj->fk_statut, 5, $obj->billed).'</td>';
+			$parameters = array('obj' => $obj);
+			$morehtmlstatus = $objectstatic->LibStatut($obj->fk_statut, 5, $obj->billed);
+			$reshook = $hookmanager->executeHooks('moreHtmlStatus', $parameters, $object); // Note that $action and $object may have been modified by hook
```

Comment:
Hook moreHtmlStatus is already used for another use case.
Also hook should be added into the LibStatut method.